src/Board.elm

```diff
@@ -0,0 +1,25 @@
+module Board exposing (..)
+
+import List exposing (repeat, take, drop, head)
+
+type alias Board = List String
+
+new : Int -> Board
+new size = 
+  repeat size ""
+
+mark : Int -> String -> Board -> Board
+mark position symbol board =
+  let
+      start = take (position - 1) board
+      end = drop position board
+      newMark = [symbol]
+  in
+      start ++ newMark ++ end
+
+markAt : Int -> Board -> String
+markAt position board =
```

Comment:
The comment here seems to be for the entire list of functions on board, but I assume you are talking about `markAt`. And yes, it's really just a test method at this point. I'm not sure what I will end up with as a data structure here. List is proving a bit tricky.